### ISSUE TITLE
Planning a forked overhaul

### ISSUE THREAD

**Issue by smilansky** [yogiben/meteor-admin#339]
Hey guys - I have been using a forked/modded implementation of this package for the past few months for a client of mine. After spending a good amount of time working with it, I wanted to solicit help with releasing a modded version, that will address the following (listed in terms of priority):

- Customized templates that work
- Customize view tables (number of columns, etc)
- Custom user table columns
- Subscriptions/publications (add in SubsManager)
- Applying role changes and scoped roles
- Flow router
- Duplicate record button

Please mention any issues or critical features that you think will make this a better admin package in this thread and let me know if you want to help. Yogiben and anyone who has contributed - thanks for making a great package!

**Comment by obiwarn** [comment]
Awesome.
I think you addressed the main concerns people have.

What could be useful is 
- an "Apply" button in edit view
- default sort columns for tables (e.g. sort by date when table opens)
- "auto columns" like in Houston admin (i.e. auto add all columns of a table if none are specified in config)

**Comment by JohnRodney** [comment]
I think an admin panel with 1.3 support using imports and maybe a registration system is going to be a growing demand now.

**Comment by obiwarn** [comment]
any news on this?

**Comment by smilansky** [comment]
@obiwarn - I was waiting to start this with 1.3 support

**Comment by obiwarn** [comment]
@smilansky: Sadly, it's crunch time here. So the team would kill me.

**Comment by JohnRodney** [comment]
@smilansky I wouldn't mind helping I know my boss is looking for an out of the box admin solution for meteor, but I had a hard time signing off on this repo until I could see it work properly with 1.3.  If you would be willing to share with me some of your planning /whiteboarding ect I may be able to find a few hours a week to help contribute to this.  You can contact me vai email at jrodney@poeticsystems.com

**Comment by JulianKingman** [comment]
Keep me in the loop on this, I'm interested.

**Comment by maxkferg** [comment]
We are interested in implementing many of the changes mentioned in this thread at [Nitrolabs](http://www.nitrolabs.com/). In particular, we want to start with the following:
- Flow router
- Import/export support for models
- Support for admin users with accounts-google
- Rich text editor support and some other customized templates

We feel that these features are needed ASAP if people are going to use this package with Meteor 1.3. If anyone has some ideas/code that could help please let us know. Either way, we will start pushing ahead in August.

**Comment by JulianKingman** [comment]
I found these, is there something you would improve on? It seems like Meteor-Factory is more recent
https://github.com/meteor-factory/meteor-admin
https://github.com/sachinbhutani/flow-db-admin/

@mpowaga are we reinventing the wheel in this discussion?

**Comment by avishaan** [comment]
@JulianKingman, are any of those properly supported, they seem like they haven't been worked on in a while

**Comment by avishaan** [comment]
@maxkferg I really need this for 1.3, at least to fix the global import issue

**Comment by maxkferg** [comment]
@codeHatcher. We have moved over to the https://github.com/meteor-factory/meteor-admin fork. It is better set up for Meteor 1.3 and ES6

**Comment by avishaan** [comment]
@maxkferg I saw that but am really hesitant to use that because it has been 7 months since the last commit. This makes me think it is no longer supported or being looked at. Is it still in (semi) active development?

**Comment by maxkferg** [comment]
@codeHatcher. I agree with you - this fork (yogiben) seems to have a lot more activity. There are three reasons why we decided to go with the mfactory fork:

1) It already supports iron-router and flow-router
2) It is written in pure js, not coffee ([js makes more sense right now](https://robots.thoughtbot.com/replace-coffeescript-with-es6))
3) Collections do not have to be global (Works much better with Meteor 1.3)

We think the mfactory fork is a better place to focus development effort, but this fork clearly has more activity. I would like to hear from @yogiben about this

**Comment by avishaan** [comment]
@maxkferg I agree with all your points. 7 months is just sooooo long in the js meteor world. I'm looking at other alternatives now including flow-db-admin and orionjs hoping they have options that work with 1.3

really your 3 point is the most important for me right now
